tests/UNMOCKED_end2end_tests/delete_datasets_test_unmocked_cli.py

```diff
@@ -12,7 +12,8 @@
 
     api_workflow_client = ApiWorkflowClient(token=token)
 
-    for i in range(num_datasets):
+    num_datasets = int(num_datasets)
+    for i in range(1, num_datasets+1):
         dataset_name = f"test_unmocked_cli_{i}"
         api_workflow_client.create_dataset(dataset_name)
```

Comment:
can you explain why we try to create the dataset with the same name you used in your prev scripts here again?

Reply:
Because of 
https://github.com/lightly-ai/lightly/blob/38fd246a9bb2a521501413eeb89c62773b8f0c10/lightly/api/api_workflow_datasets.py#L31-L34

But you are right, there is a function doing this directly.